src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CaseCreationResult.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.events.CaseCreationResultType;
+
+public class CaseCreationResult {
+
+    public final CaseCreationResultType resultType;
+    public final Long caseCcdId;
+
+    public CaseCreationResult(CaseCreationResultType resultType, Long caseCcdId) {
```

Comment:
looks like can be private. and class marked as final

Reply:
true, will change

Reply:
done